ocaml/testsuite/tests/typing-immediate/immediate.ml

```diff
@@ -149,6 +148,9 @@ Error: The layout of type string is value, because
        But the layout of type string must be a sublayout of immediate, because
          of the definition of t at line 2, characters 2-31.
 |}];;
+(* CR layouts v2.9: The "of the definition of t ..." part is not great and it
```

Comment:
Are there also internal tickets about these? Just leaving CRs makes it possible we'll forget.

Reply:
Made tickets to track both CRs